### PR TITLE
Update Kafka sink params

### LINKED ISSUE #1346: Document: fix: allow configure message.timeout.ms and max.in.flight for kafka sink
This issue tracks the documentation update needed for the merged PR #12574.

Source PR URL: https://github.com/risingwavelabs/risingwave/pull/12574
Source PR Merged At: 2023-09-28T11:15:01Z

### PR DESCRIPTION
**Pull request by emile-00**
## Info

- **Description**

  Doc parameters `message.timeout.ms` and `max.in.flight.requests.per.connection` for Kafka sink.

- **Notes**

  [ Include any supplementary context or references here. ]

- **Related code PR**

  https://github.com/risingwavelabs/risingwave/pull/12574

- **Related doc issue**
  
  Resolves https://github.com/risingwavelabs/risingwave-docs/issues/1346

## For reviewers

- **Preview**

  [ Paste the preview link to the updated page(s) here. Edit this item after the preview site is ready. To find the updated pages, scroll down to locate and open the Amplify preview link and select the **upcoming** version of the documentation. ]

- **Key points**

  [ Parts that may need revision or extra consideration. ]

## Before merging

- [ ] I have checked the doc site preview, and the updated parts look good.

- [ ] I have acquired the approval from the owner (and optionally the reviewers) of the code PR and at least one tech writer (`CharlieSYH`, `emile-00`, & `hengm3467`).

### CODE REVIEW

**Comment by emile-00** [comment]
@hzxa21 Please take some time to review the doc updates, thank you!

File: docs/guides/create-sink-kafka.md [hunk 1]
```diff
@@ -50,6 +50,8 @@ All `WITH` options are required unless explicitly mentioned as optional.
 |AS select_query| A SELECT query that specifies the data to be output to the sink. Either this query or a FROM clause must be specified. See [SELECT](/sql/commands/sql-select.md) for the syntax and examples of the SELECT command.|
 |`connector`| Sink connector type must be `'kafka'` for Kafka sink. |
 |`properties.bootstrap.server`|Address of the Kafka broker. Format: `‘ip:port’`. If there are multiple brokers, separate them with commas. |
+|`max.in.flight.requests.per.connection`|Optional. The maximum number of unacknowledged requests the Kafka client sends on a connection before blocking. By default, this value is 5. |
+|`message.timeout.ms`| Optional. The amount of time, in milliseconds, a produced message waits for delivery before timing out. By default, this value is 5000. |
```
Comment (hzxa21) [inline docs/guides/create-sink-kafka.md]: These are kafka client configurations so I think it is better to put it in "[Additional Kafka Parameters](https://github.com/risingwavelabs/risingwave-docs/blob/cf0403069dd7e7127bca6e678aadd4a35a71e7ad/docs/guides/create-sink-kafka.md#additional-kafka-parameters)" section.

File: versioned_docs/version-1.3/guides/create-sink-kafka.md [hunk 2]
```diff
@@ -50,6 +50,8 @@ All `WITH` options are required unless explicitly mentioned as optional.
 |AS select_query| A SELECT query that specifies the data to be output to the sink. Either this query or a FROM clause must be specified. See [SELECT](/sql/commands/sql-select.md) for the syntax and examples of the SELECT command.|
 |`connector`| Sink connector type must be `'kafka'` for Kafka sink. |
 |`properties.bootstrap.server`|Address of the Kafka broker. Format: `‘ip:port’`. If there are multiple brokers, separate them with commas. |
+|`max.in.flight.requests.per.connection`|Optional. The maximum number of unacknowledged requests the Kafka client sends on a connection before blocking. By default, this value is 5. |
+|`message.timeout.ms`| Optional. The amount of time, in milliseconds, a produced message waits for delivery before timing out. By default, this value is 5000. |
```
Comment (hzxa21) [inline versioned_docs/version-1.3/guides/create-sink-kafka.md]: ditto

**Review by hzxa21 — APPROVED** [review]
LGTM

**Pull request merged**